bin/lib/zopen-build

```diff
@@ -204,7 +221,7 @@ printSyntax()
   echo "  -vv: run in very verbose mode (sets environment variables V=1 and VERBOSE=1)"
   echo "  -u|--upgradedeps: upgrade all dependencies by running zopen install"
   echo "  --buildtype: release|debug. The default is release"
-  echo "  --type: tarball|git|bare."
+  echo "  --build: dev|stable. This is the build level to build off of. If no build level is specified, it will use the ZOPEN_BUILD_LEVEL specified in the build, otherwise it will fallback to ZOPEN_TYPE."
```

Comment:
Are ZOPEN_TYPE, ZOPEN_TARBALL_*, ZOPEN_GIT_* still active? Is ZOPEN_TYPE no longer _required_ ? 
Should we mark those env vars as 'deprecated' with the intent that new packages shouldn't use them?
Should your tool for creating a new package be updated to use these new env vars?